libraries/app/include/graphene/app/api.hpp

```diff
@@ -298,8 +298,28 @@ namespace graphene { namespace app {
           */
          std::vector<net::potential_peer_record> get_potential_peers() const;
 
+         /**
+          * @brief Return list of pending transactions.
+          */
+         std::vector<signed_transaction> list_pending_transactions() const;
```

Comment:
I'd rather put these API's in `network_broadcast_api` or another, because `network_node_api` is for peers and is usually hidden from public due to security reasons.

Reply:
I wanted it in network because i want it restricted ... you belive it is safe to have it "open" to everyone?

Reply:
Right now, I more want to focus on public API's. Restricted API's, if not for node monitoring or performance analysis, are less valuable. Pending transactions IMHO is OK to be public.